main/modes/platformer/mode_platformer.c

```diff
@@ -566,8 +541,8 @@ void updateGameClear(platformer_t *self){
 
 void drawGameClear(display_t *d, font_t *font, gameData_t *gameData){
     drawPlatformerHud(d, font, gameData);
-    drawText(d, font, c555, "Thanks for playing.", 24, 32);
-    drawText(d, font, c555, "Many more battle scenes", 8, 80);
-    drawText(d, font, c555, "will soon be available!", 8, 96);
-    drawText(d, font, c555, "Bonus 100000pts per life!", 8, 144);
+    drawText(d, font, c555, "Thanks for playing.", 24, 48);
```

Comment:
Same note about centering text. You can also use a `yIdx` variable draw the text neatly, so start it at `48` and add `font->h + 4` after each line is drawn. That way you could swap fonts and keep a nice layout.